src/detect-env.ts

```diff
@@ -1,7 +1,8 @@
 import { Env, type EnvDetails } from './env.types'
 
 export interface DetectEnvContext {
-  syntheticEventDetected: boolean
+  syntheticEventDetected?: boolean
```

Comment:
That would be removed once `<FpjsProvider />` starts using `WithEnvironment` under the hood.